webpack/storybook/atoms/UploaderField.stories.js

```
@@ -0,0 +1,172 @@
+WithPreviewValidations.args = {
+  uploadedFiles: [],
+  variant: "uploadField",
+  showAsPreview: true,
+  multiple: true,
+  validations: {
+    maxSizeAllowedInMB: 5,
+    minSizeRequiredInMB: 1,
+  },
+};
+
+export const Error = Template.bind({});
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)